### ISSUE TITLE
angular.io missing from google results for certain users/devices

### ISSUE THREAD

**Issue by IgorMinar** [angular/angular#21813]
I was able to repro this only on my mobile phone but @StephenFluin and @bradlygreen reported seeing it on their desktop too:

![screenshot_20180126-141646](https://user-images.githubusercontent.com/216296/35463135-101f37a4-02a4-11e8-9e56-916d07996e95.png)
![screenshot_20180126-141638](https://user-images.githubusercontent.com/216296/35463134-0fff61ea-02a4-11e8-96c3-56be18ef3967.png)

**Comment by IgorMinar** [comment]
I've checked crawler stats and they look fine as far as I can tell. I also tried fetching the site as robot and it succeeded:

![screen shot 2018-01-26 at 2 17 18 pm](https://user-images.githubusercontent.com/216296/35463190-45b54f20-02a4-11e8-9ce9-f6906088853f.png)
![screen shot 2018-01-26 at 2 17 27 pm](https://user-images.githubusercontent.com/216296/35463191-45cfe8b2-02a4-11e8-8017-48e636be9fee.png)

**Comment by IgorMinar** [comment]
I think we'll just monitor it for a bit first while watching analytics / crawler stats to understand the impact. It's unclear to me what could be responsible for this.

The only somewhat relevant changes we made recently were: #21665 and #21689.

There there are #21712 #21763 but those didn't make it to prod yet.

**Comment by IgorMinar** [comment]
I also got a notification that we now have new search console available for our site, but that seems unrelated.

**Comment by IgorMinar** [comment]
The index status is looking fine:

![screen shot 2018-01-26 at 2 33 24 pm](https://user-images.githubusercontent.com/216296/35463531-ee71fee6-02a5-11e8-877e-0feec2e7b525.png)

**Comment by IgorMinar** [comment]
Sessions are 4-20% down for today (ignore the sharp drop - we don't have complete data for today yet)
![screen shot 2018-01-26 at 2 45 19 pm](https://user-images.githubusercontent.com/216296/35463870-c63242ae-02a7-11e8-8c63-32a49fc104cc.png)

sessions were already down by 4% down at 6am:
![screen shot 2018-01-26 at 2 45 54 pm](https://user-images.githubusercontent.com/216296/35463871-c6811302-02a7-11e8-810d-0ac4605a5e6a.png)

But the number of users is down sharply - 20-60% for today:
![screen shot 2018-01-26 at 2 46 17 pm](https://user-images.githubusercontent.com/216296/35463874-c6cc8c1a-02a7-11e8-8ad1-805893621e17.png)

Started around 7pm PT yesterday:
![screen shot 2018-01-26 at 2 46 44 pm](https://user-images.githubusercontent.com/216296/35463875-c714e5b4-02a7-11e8-9225-950802f272b6.png)

**Comment by softplus** [comment]
It looks like these pages have a Googlebot noindex meta tag on them -- this prevents Google from indexing the page. Using JavaScript to remove limitations like these is not recommended, and can result in those pages being dropped from indexing before they're rendered.

**Comment by IgorMinar** [comment]
Thanks @softplus. In that case the culprit PR is #21665. I find it however very odd that the search console didn't warn us about this.

**Comment by petebacondarwin** [comment]
So we should remove the pre-emptive hard coded tag?

**Comment by IgorMinar** [comment]
@petebacondarwin yeah. I have a PR: #21816 out for review

**Comment by IgorMinar** [comment]
The fix in #21816 is now in production.

**Comment by petebacondarwin** [comment]
@IgorMinar - Is this now resolved?

**Comment by IgorMinar** [comment]
I'm waiting for more analytics data (it's all delayed), but yesterday
everything looked good.

**Comment by IgorMinar** [comment]
looks like we recovered.

and interestingly, I just got an alert from google that we were doing it wrong. I wish it didn't take 9 days for these alerts to arrive:

![screenshot 2018-02-04 at 3 25 01 pm](https://user-images.githubusercontent.com/216296/35783723-dd16dfbe-09c0-11e8-9d98-1095f03180ae.png)

![screenshot 2018-02-04 at 3 26 49 pm](https://user-images.githubusercontent.com/216296/35783726-e18b29a6-09c0-11e8-9526-cb0b17453ea7.png)

note that the date when the crawling errors occurred are around the same time as we noticed the the decreased traffic from google.com results.
